pkg/storage/query.go

```diff
@@ -292,16 +292,36 @@ func convertToRawJsonDocument(in interface{}, raw interface{}) error {
 // * matching namespace
 // * name contains substring
 // * labels contains all matches
-func CreateListFiler(namespace string, name string, labels map[string]string) map[string]interface{} {
+// func CreateListFiler(namespace string, name string, labels map[string]string) map[string]interface{} {
```

Comment:
Can you remove this commented out function signature?

Reply:
Ah, sorry that this got left out. Thank you for pointing this out!